test/dhcp/server_ms_test.rb

```diff
@@ -14,140 +14,12 @@ def setup
-    172.29.205.89     -    00-21-28-57-40-e2-
-    172.29.205.90     -    00-14-4f-ca-6a-70-
-    172.29.205.70     -    00-21-28-57-41-72-
-    172.29.205.31     -    99-10-10-11-12-12-
-    172.29.205.10     -    00-1b-24-1d-e9-e4-
-    172.29.205.57     -    00-14-4f-41-0a-18-
-    172.29.205.76     -    00-e0-81-bb-05-18-
-    172.29.205.71     -    00-14-4f-ca-91-48-
-    172.29.205.91     -    00-e0-81-d2-ea-12-
-    172.29.205.92     -    00-e0-81-d2-ea-06-
-    172.29.205.93     -    00-e0-81-d2-01-aa-
-    172.29.205.94     -    00-e0-81-d1-fe-2a-
-    172.29.205.95     -    00-e0-81-d2-e9-aa-
-    172.29.205.20     -    00-0c-29-e4-cd-0c-
-    172.29.205.21     -    00-0c-29-3b-ec-dc-
-    172.29.205.22     -    00-0c-29-cf-00-45-
-    172.29.205.24     -    00-0c-29-e9-83-09-
-    172.29.205.96     -    00-e0-81-d2-e9-56-
-    172.29.205.6      -    00-14-4f-82-9c-ea-
-    172.29.205.27     -    00-03-ba-45-79-31-
-    172.29.205.23     -    00-0c-29-f6-f9-ce-
-    172.29.205.25     -    00-0c-29-64-a1-f1-
-    172.29.205.26     -    00-0c-29-46-8f-10-
-
-
-No of ReservedIPs : 57 in the Scope : 172.29.205.0.
-
-Command completed successfully.
-')
-    @server.stubs(:execute).with("scope 172.29.205.0 Show OptionValue", "Queried 172.29.205.0 options").returns('
-Changed the current scope context to 172.29.205.0 scope.
-
-Options for Scope 172.29.205.0:
-
-        DHCP Standard Options :
-        General Option Values:
-        OptionId : 81
-        Option Value:
-                Number of Option Elements = 1
-                Option Element Type = DWORD
-                Option Element Value = 0
-        OptionId : 51
-        Option Value:
-                Number of Option Elements = 1
-                Option Element Type = DWORD
-                Option Element Value = 691200
-        OptionId : 3
-        Option Value:
-                Number of Option Elements = 1
-                Option Element Type = IPADDRESS
-                Option Element Value = 172.29.205.1
-Command completed successfully.
-    ')
+    @server.stubs(:execute).with("show scope", "Enumerated the scopes on 1.2.3.4").returns(["", "==============================================================================", " Scope Address  - Subnet Mask    - State        - Scope Name          -  Comment", "==============================================================================", " 172.24.166.0   - 255.255.255.128-Active        -WLAN SMW Clients BRS -WLAN range SMW", " 172.24.230.0   - 255.255.255.128-Active        -WLAN SMW Clients BRS -WLAN range SMW", " 172.29.204.0   - 255.255.255.0  -Active        -Client VLAN Scope    -DC Standardiza", " 172.29.205.0   - 255.255.255.128-Active        -Server VLAN Scope    -DC Standardiza", " 172.29.205.128 - 255.255.255.128-Active        -Management VLAN Scope-DC Standardiza", " 172.29.216.0   - 255.255.254.0  -Active        -DC BRS               -", " Total No. of Scopes = 6", "Command completed successfully."])
```

Comment:
This is pretty hard to read. What I had in mind was to keep existing multi-line strings around, but to split them on new-lines...